chrome-extension/background.js

```diff
@@ -1,12 +1,83 @@
-chrome.runtime.onInstalled.addListener(function() {
-  chrome.declarativeContent.onPageChanged.removeRules(undefined, function() { // undefined removes all rules
+const MS_BEFORE_CLEAR_NOTIFICAITON = 2000;
+const VIBE_CHECK_MENU_ID = 'vibe check';
+const TOXICITY_CHECK_MENU_ID = 'toxicity check';
+
+chrome.runtime.onInstalled.addListener(function () {
+  chrome.declarativeContent.onPageChanged.removeRules(undefined, function () { // undefined removes all rules
     chrome.declarativeContent.onPageChanged.addRules([{
       // required conditions for the extension to be active
       conditions: [new chrome.declarativeContent.PageStateMatcher({
-        pageUrl: {hostEquals: 'www.youtube.com'},
+        pageUrl: { hostEquals: 'www.youtube.com' },
       })],
       // things to do if the extension is active
       actions: [new chrome.declarativeContent.ShowPageAction()]
     }]);
   });
 });
+
+chrome.runtime.onInstalled.addListener(function () {
+  const createProperties = {
+    'id': VIBE_CHECK_MENU_ID,
+    'title': 'Vibe check this video',
+    'contexts': ['link'],
+  };
+
+  chrome.contextMenus.create(createProperties);
+});
+
+chrome.runtime.onInstalled.addListener(function () {
+  const createProperties = {
+    'id': TOXICITY_CHECK_MENU_ID,
+    'title': 'Toxicity check this line',
+    'contexts': ['selection'],
+  };
+
+  chrome.contextMenus.create(createProperties);
+});
+
+chrome.contextMenus.onClicked.addListener(async function (info) {
+  if (info.menuItemId == VIBE_CHECK_MENU_ID) {
+    const notificationOption = {
+      type: 'basic',
+      iconUrl: '/images/temp-icon.png',
```

Comment:
Nit: Put this string into a constant above so it could be changed in the future easily

Reply:
done